Workbooks/KeyVault/Azure Monitor Onboarding/Azure Monitor Onboarding.workbook

```diff
@@ -0,0 +1,281 @@
+          }
+        ],
+        "style": "above",
+        "queryType": 1,
+        "resourceType": "microsoft.operationalinsights/workspaces"
+      },
+      "name": "KeyVault-Parameters",
+      "styleSettings": {
+        "margin": "15px 0 0 0"
+      }
+    },
+    {
+      "type": 3,
+      "content": {
+        "version": "KqlItem/1.0",
+        "query": "let onboardedResources = union withsource=TableName *\r\n| where TableName == 'AzureDiagnostics'\r\n| extend ResourceId = tolower(column_ifexists('ResourceId', ''))\r\n| summarize by ResourceId;\r\nlet data = range i from 1 to 1 step 1\r\n| project resource = todynamic(tolower(strcat('[', '{Vaults}', ']')))\r\n| mvexpand resource to typeof(string) limit 10000\r\n| join kind = fullouter (onboardedResources) on $left.resource == $right.ResourceId;\r\ndata\r\n| summarize Resources = countif(ResourceId != '')\r\n| extend Rank = 1, Title = 'Enabled'\r\n| union (data\r\n| summarize Resources = countif(ResourceId == '')\r\n| extend Rank = 2, Title = 'Disabled', Footer = 'Change')\r\n| order by Rank asc",
+        "size": 4,
+        "title": "Key vaults with diagnostic logs collection",
+        "timeContext": {
+          "durationMs": 86400000
+        },
+        "exportToExcelOptions": "visible",
+        "queryType": 0,
+        "resourceType": "microsoft.keyvault/vaults",
+        "crossComponentResources": [
+          "{Vaults}"
+        ],
+        "visualization": "tiles",
+        "tileSettings": {
+          "titleContent": {
+            "columnMatch": "Title",
+            "formatter": 1,
+            "formatOptions": {
+              "showIcon": true
+            }
+          },
+          "leftContent": {
+            "columnMatch": "Resources",
+            "formatter": 12,
+            "formatOptions": {
+              "showIcon": true
+            },
+            "numberFormat": {
+              "unit": 17,
+              "options": {
+                "style": "decimal",
+                "maximumFractionDigits": 2,
+                "maximumSignificantDigits": 3
+              }
+            }
+          },
+          "rightContent": {
+            "columnMatch": "Title",
+            "formatter": 18,
+            "formatOptions": {
+              "showIcon": true,
+              "thresholdsOptions": "icons",
+              "thresholdsGrid": [
+                {
+                  "operator": "==",
+                  "thresholdValue": "Disabled",
+                  "representation": "2",
+                  "text": ""
+                },
+                {
+                  "operator": "Default",
+                  "thresholdValue": null,
+                  "representation": "success",
+                  "text": ""
+                }
+              ]
+            }
+          },
+          "secondaryContent": {
+            "columnMatch": "Footer",
+            "formatter": 1,
+            "formatOptions": {
+              "linkTarget": "Url",
```

Comment:
Just to make sure - this makes a 'Change' link show on the tile which goes nowhere for me.
It will work once deployed?